plugins/repository-hdfs/build.gradle

```diff
@@ -156,31 +159,21 @@ if (fixtureSupported) {
 boolean secureFixtureSupported = false;
 if (fixtureSupported) {
   // Only do secure fixture support if the regular fixture is supported,
-  // and if vagrant is installed. The ignoreExitValue on exec only matters
-  // in cases where the command can be found and successfully started. In
-  // situations where the vagrant command isn't able to be started at all
-  // (it's not installed) then Gradle still throws ExecException.
-  ByteArrayOutputStream pipe = new ByteArrayOutputStream()
-  try {
-    ExecResult runResult = exec {
-      commandLine 'vagrant', '--version'
-      standardOutput pipe
-      ignoreExitValue true
+  // and if vagrant is supported.
+  if (project.vagrantSupported == false) {
+    // Log the reason
+    if (project.vagrantInstallation.supported == false) {
+      logger.warn(project.vagrantInstallation.error.message)
     }
-    String output = pipe.toString().trim()
-    if (runResult.exitValue == 0) {
-      secureFixtureSupported = (output ==~ /Vagrant 1\.(8\.[6-9]|9\.[0-9])+/)
-    } else {
-      logger.warn("Could not read installed vagrant version:\n" + output)
+    if (project.virtualBoxInstallation.supported == false) {
+      logger.warn(project.virtualBoxInstallation.error.message)
```

Comment:
These messages are becoming noisy, and aren't actually helpful, because they are emitted even when not trying to run this task. I wonder if instead you could check emit this at runtime if the task would have run? You could do this as a doFirst, and throw a StopExecutionException if the task should be skipped.